### DISCUSSION TITLE
Adding playing with bot or only bot mode feature

### DISCUSSION THREAD

**Discussion by Skyfuzzball8312** [ekiefl/pooltool#195] [Ideas]
Here's how it's work
A bot's stats consists of:

- Power: The higher this stat is, the higher the maximum power allowed shot.
- Jump: The higher this stat is, the better the jump.
- Spin: The higher this stat is, the better the spins.
- Curve: The higher this stat is, the better the curve.

Note that: Each stats must have at 3 sub-stats: The Main Stats, The Technical Stats and The consistency Stats (E.g: Power, Power Precision, Power Consistency)

Player can create bot which each stats must have the average overall rating no more than 100 points.
Player can select the difficulty of the bot by entering OVR modifier: -10 is easiest, +10 is hardest.

**Comment by ekiefl** [comment]
Thanks for your idea. This is in line with some loosely planned future developments. No ETA or roadmap yet though.

What does OVR mean?

**Comment by Skyfuzzball8312** [comment]
OVR Stands for Overall Points

Additional Comment:
- Also, player can create cue with 4 stats: Aiming Object Lines, Cushion Object Lines (If Aiming Lines are enabled), Power and Spins, and each stats can be under or over 100 points, usually a cue's overall rating must equal to 25
